libraries/ArduinoOTA/examples/OTAWebUpdater/OTAWebUpdater.ino

```diff
@@ -0,0 +1,160 @@
+#include <WiFi.h>
+#include <WiFiClient.h>
+#include <ESP32WebServer.h>
```

Comment:
it is just WebServer. No ESP32 in front

Reply:
yes, WebServer is required for Zero config routing protocol (http://hostname.local), just to detect your device in LAN connectivity. if WebServer not used then everytime the user has to detect Ip of esp32 
And eventually, Firmware will be uploaded in esp32 only.

Reply:
Yes... but the name of the lib is not ESP32WebServer, but just WebServer ;) I get why it's needed :D

Reply:
https://github.com/espressif/arduino-esp32/blob/master/libraries/WebServer/src/WebServer.h